src/main.rs

```diff
@@ -6,6 +6,11 @@ mod commands;
 mod errors;
 mod utils;
 
+// use jemalloc for release builds
+extern crate jemallocator;
+#[global_allocator]
+static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
+
```

Comment:
i'm unfamiliar with `jemallocator`. can you give a tl;dr on these performance optimizations? what do they do?

Reply:
Is there a way to only use my first commit in the PR? The second commit is performance optimized stuff that only belongs on my fork.

TL;DR this can be removed for the main branch